frontend/src/styles.scss

```diff
@@ -246,3 +246,21 @@ body {
 .ant-notification-notice-message {
 	padding-right: 20px;
 }
+
+.ant-drawer {
+	// do not modify this as this is required to hide intercom in modals
+	z-index: 1000000000000;
```

Comment:
???

Reply:
this is to hide intercom modals when drawer is open. the z-index for intercom modal is huge. this is done to surpass that